src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -152,6 +152,16 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    handleAnnotationEdit = (id: string, text: string, permissions: AnnotationPermission) => {
+        const { api, file } = this.props;
+        const errorCallback = (e, code) => this.feedErrorCallback(e, code);
+        const successCallback = () => this.feedSuccessCallback();
+
+        api.getFeedAPI(false).updateAnnotation(file, id, text, permissions, successCallback, errorCallback);
```

Comment:
Can we inline the callbacks like `api.getFeedAPI(false).updateAnnotation(file, id, text, permissions, this.feedSuccessCallback, this.feedErrorCallback);`

Reply:
Yep that works.